providers/github/connection/platform.go

```diff
@@ -142,8 +149,7 @@ func (c *GithubConnection) Repository() (*github.Repository, error) {
 
 	repoId := conf.Options["repository"]
 	if ownerId != "" && repoId != "" {
-		repo, _, err := c.Client().Repositories.Get(context.Background(), ownerId, repoId)
```

Comment:
is there a place where we verify the data?

Reply:
not sure what you mean by this. We don't do in memory validation. We just call the github api eventually with this info